configs/raspberrypi_config

```diff
@@ -92,11 +92,24 @@ function platform_modify() {
         run_and_log_cmd "sudo ln -fs /uboot/${f} work/rootfs/boot/${f}"
     done
 
-    # Raspberry Pi headless configuration expects boot partition to be mounted
-    # at /boot, so replace these services with /uboot
-    run_and_log_cmd "find work/rootfs/lib/ -type f -name *.service | xargs sed -i.bak 's|/boot/|/uboot/|g'"
-    log_debug "Diff from modified service files:\n$(find work/rootfs/lib/ -type f -name '*.service' | xargs -I@ bash -c 'diff -u @.bak @')"
-    run_and_log_cmd "find work/rootfs/lib/ -type f -name *.service.bak | xargs rm"
+    # Raspberry Pi headless configuration and other first boot scripts inspect the boot partition
+    # for files. Modify the hard-coded mount point for the partition from /boot to /uboot.
+
+    # Integrate with systemd services with "/boot/" hardcoded
+    files_to_modify=$(find work/rootfs/lib/ -type f -name *.service | xargs grep '/boot/' | cut -d: -f1 | uniq)
```

Comment:
Q: don't we have to quote the shell wildcard `_*_` here? :thinking:

Reply:
Yes, we should.

It only matters if there is a file `*.service` in the working directory but as a general rule we should do it to be safe. I'll add it, thanks!